docs/software-platforms/aws/starting-development-on-lsfr.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with development on Lightsail for Research
+
+Once you have [created a virtual computer](creating-vcs.md), [attached a disk](working-with-volumes.md), and [accessed the operating system](creating-vcs.md#how-to-access-a-virtual-computer) you are ready to start developing analyses on Lightsail for Research.
+
+## Quick tips about virtual computers
+
+- Copy and paste need to be synced between your local and remote desktop using the clipboard icon in the upper left hand corner.
+
+    <figure markdown="span">
+        ![Clipboard](../../img/lsfr-clipboard.png){width="400"}
+    </figure>
+
+    You will also need to right-click to paste on the virtual desktop; `Ctrl+v` or `Cmd+v` will not work.
+
+- If you use a security key as a multifactor authentication method for GitHub, that will not work on a virtual computer.
+Make sure you have another way to login with multifactor authentication, such as an authenticator app.
+
+## Clone the repository
+
+Virtual computers will have Git and GitKraken installed on them.
+Click `Activities` on the upper left hand corner of the Ubuntu desktop then click the 9-dot menu to access the applications on the virtual computer.
+
+<figure markdown="span">
+    ![Activities](../../img/lsfr-activities.png){width="800"}
+</figure>
+
+<figure markdown="span">
+    ![Candy box menu](../../img/lsfr-candy-box-menu.png){width="800"}
+</figure>
+
+Open GitKraken and login using GitHub.
+
+Clone the repository to the volume you created and attached following [the setup instructions for cloning the repository](../../technical-setup/clone-the-repo.md#clone-your-forked-repository).
+
+!!! tip "Using the Git CLI"
+    If you prefer to use the command line interface, you can locate and open the Terminal using the same steps (`Activities` followed by the menu).
+
+## Create and activate a conda environment
+
+Conda will be installed on virtual computers, but no setup has been performed.
+
+First, add channels with the following command:
+
+```sh
+conda config --add channels defaults
+conda config --add channels bioconda
+conda config --add channels conda-forge
+conda config --set channel_priority strict
+```
+
+
+You will not have permission to install programs into your `base` environment as described in [the conda setup documentation](../../technical-setup/environment-setup/setup-conda.md).
+
+Instead, you need to first create a conda environment using Terminal as follows:
+
+```sh
+conda create --name openscpca
+```
+
+Then, activate the environment with the following command:
+
+```sh
+conda activate openscpca
+```
+
+Now you can install [the minimal conda environment](../../technical-setup/environment-setup/setup-conda.md#set-up-conda) with the following command:
+
+```sh
+conda install awscli conda-lock jq pre-commit
```

Comment:
Is there a reason we don't want them to just do all of this in one command with `conda env create -f environment.yml`. We provide all of these packages in the yml file?

Reply:
I don't know, and reviewing #251 didn't clear things up for me. It seems better to be consistent.

Reply:
I think the idea with the local install was that people might want to just put these in their base environment and not have to create a separate environment that they have to remember to activate to interact with the repo/aws. But for here, since they have to make the environment anyway, I would do it all with the `conda env create -f environment.yml`

Reply:
+1 to Josh's comment